src/compas/geometry/geometry.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from compas.data import Data
+
+
+class Geometry(Data):
+    """Base class for all geometric objects."""
+
+    def __ne__(self, other):
+        # this is not obvious to ironpython
```

Comment:
😆  +1 for moar of these in code

Reply:
is an old one by @beverlylytle if i remember correctly :)